docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,74 +506,104 @@ If you submit a ledger API command (which can have multiple Daml commands), then
 
 Just putting all Daml commands into a single batch command does not help, as you might still create lots of views.
 
-In the protocol up to version 5, Canton creates a view for every action node in the transaction tree if the informees change 
-between the node and its parent. Therefore, in order to minimize the number of views, you should try to minimize the number of 
-changes of informees. Informees are roughly the signatories, observers of the contract and of the choice, and the controllers of the choice. 
+Strategy for Protocol Version 6
++++++++++++++++++++++++++++++++
 
-Instead of sending a batch with 20 exercise commands `Foo`, group your contracts by stakeholder and send one exercise 
-command on a generator contract with the list of contracts that in turn exercises the `Foo` choice per stakeholder group.
+Starting from protocol version 6, Canton creates a view for every action node in the transaction tree if the
+**participants that host their informees are not a subset of their parent view's informee participants**. Therefore, to
+reduce the number of views, you should try to reduce either the number of times the set of informees grows or
+make sure most of your actions share the same pool of participants that must be informed. Alternatively, you can
+add informees to the action nodes close to the root (e.g., by including choice observers) so that their children nodes
+are aggregated in the initial view, since these children nodes will only target a subset of those participants.
 
-If you write batch commands, group the operations on the ledger based on the informees and create a single choice and 
-batch operation per informee group. You can use choice observers to align informees if necessary. 
+Instead of sending a batch with 20 exercise commands `Foo`, group your contracts by a set of
+stakeholders that cover the exercise's informee participants and send one exercise command on a generator contract
+with the list of contracts that in turn exercises the Foo choice per stakeholder group.
 
-For example, if the below `doUpdate` is only visible to the `owner` party, then whether the `efficient` flag is set or not 
+In general, if you write batch commands, group the operations on the ledger based on the informees' participants and create
+a single choice and batch operation per informee's participants group.
+
+For example, if the `doUpdate` is only visible to the `owner` party, then whether the `efficient` flag is set or not
 will have a huge impact on the number of views created.
 
 .. code:: daml
```

Comment:
Given the many syntactic errors in this example, it would be IMO good to source this example from a Daml file that is actually compiled as part of CI.

Reply:
I agree, I've opened an issue and will do this in a subsequent [PR](https://github.com/DACH-NY/canton/issues/20047).